src/HypLSP7Collateral.sol

```diff
@@ -15,6 +18,7 @@ contract HypLSP7Collateral is TokenRouter {
      * @param lsp7_ Address of the token to keep as collateral
      */
     constructor(address lsp7_, address mailbox_) TokenRouter(mailbox_) {
+        require(Address.isContract(lsp7_), "HypLSP7Collateral: invalid token");
```

Comment:
nice

Reply:
It was added in the hyperlane monorepo, in the latest version of their contracts